gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -460,13 +463,17 @@ end
 -- @return boolean success?
 -- @realm server
 function plymeta:GiveEquipmentItem(id)
```

Comment:
What's the difference between ply:GiveEquipmentItem and ply:AddEquipmentItem?

Reply:
Very good question. I was confused too, since they are basically the same with the same checks. But they are not done by me and maybe something relies on them. Therefore I did not want to change anything inside them

Reply:
We should give a look again to this